<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup_page</class>
 <widget class="QDialog" name="Signup_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Busbee</string>
  </property>
  <widget class="QLabel" name="logolabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resourse.qrc">:/img/images/BusBee_sgnin.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_txxt1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
color: grey;</string>
   </property>
   <property name="text">
    <string>*All Fields Are Mandatory*</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fname">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>First Name  :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Fname">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_lname">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Last Name  :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Lname">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Email          :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Email">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_phone">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Phone No   :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pw">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Password    :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Uname">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: black;</string>
   </property>
   <property name="text">
    <string>Username   :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Uname">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pw">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_txxt2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
color: grey;</string>
   </property>
   <property name="text">
    <string>By signing up, you agree to our Terms , Privacy Policy and Cookies Policy .

</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_signup">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249,191,81);
font: 700 10pt &quot;Segoe UI&quot;;
color: black;
border-radius: 20px;
</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_goback">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249,191,81);
font: 700 10pt &quot;Segoe UI&quot;;
color: black;
border-radius: 20px;
</string>
   </property>
   <property name="text">
    <string>Go to login</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resourse.qrc"/>
 </resources>
 <connections/>
</ui>
